client/v2/autocli/flag/map.go

```diff
@@ -0,0 +1,245 @@
+package flag
+
+import (
+	"context"
+	"fmt"
```

Comment:
Please separate and group:
* standard library imports
* non-cosmos-sdk imports
* cosmos-sdk imports

each separated by a newline